src/model/BoxPlotColumn.ts

```diff
@@ -0,0 +1,136 @@
+/**
+ * Created by bikramkawan on 24/11/2016.
+ */
+import * as d3 from 'd3';
+import ValueColumn from './ValueColumn';
+import Column from './Column';
+import {IDataStat} from './MultiValueColumn';
+
+
+function numSort(a, b) {
+
+  return a - b;
+}
+
+export enum Sort {
+
+  min, max, median, q1, q3, mean
+}
+
+
+export interface IBoxPlotColumn {
+  getBoxPlotData(row: any, index: number, scale: any): IBoxPlotData;
+  getDataInfo(): IDataStat;
+  getUserSortBy(): string;
+
+}
+
+interface IBoxPlotDataStat extends IDataStat {
+  min: number,
+  max: number,
+  readonly sort: string
```

Comment:
this is used to the current sorting method in renderer to draw red line to indicate what is the current sorting criteria
![image](https://cloud.githubusercontent.com/assets/12292333/21645455/4ef37c30-d292-11e6-8ed1-2fa0577ede0c.png)